docs/proposals/scheduling/aggregate-by-priority/aggregate-by-priority.md

```diff
@@ -0,0 +1,283 @@
+
+For example, the user has multiple clusters, and he wants to use some preferred clusters first.
+Only if these preferred clusters are not sufficiently allocated, the remaining clusters are used.
+
+From the perspective of a deployment, it should be like this:
+* When first propagation, prioritize allocation to preferred clusters as much as possible.
+* When replicas scaling up, prioritize replicas expansion in preferred clusters (existing replicas remain unchanged).
+* When replicas scaling down, prioritize replicas reduction in non-preferred clusters.
+
+### Goals
+
+<!-- 本文旨在提供一种调度策略，让用户可以自定义集群的优先级，让副本优先分配到优先级高的集群 -->
+
+* Aims to providing a scheduling strategy that allows users to customize the priority of clusters, 
+  ensuring that replicas are preferentially allocated to higher-priority clusters.
+
+### Non-Goals
+
+<!-- 
+用户对相同优先级集群怎么分配没有特殊诉求，合理即可，
+本文对相同优先级集群采取聚合的策略分配(将副本划分到尽可能少的集群)，不过多发散到其他复杂场景 
+-->
+
+* Users have no specific requests for allocating clusters with the same priority, as long as it is reasonable. 
+  Here uses an `Aggregated` strategy to treat that case, dividing replicas into as few clusters as possible, avoiding more complex scenarios.
+
+## Proposal
+
+### User Stories (Optional)
+
+#### Story 1
+
+As a cluster administrator, I manage two clusters, one is a cheap private cloud cluster and the other is an expensive public cloud cluster.
+
+I hope that the user's replicas will give priority to the specified cheaper cluster, whether it is first propagation or scaling up.
+Only when the cheaper cluster has insufficient capacity, the expensive cluster will be used, so that I can save more costs.
+
+### Notes/Constraints/Caveats (Optional)
+
+### Risks and Mitigations
+
+## Design Details
+
+<!-- 
+本段落提出一种名为 PriorityAggregated 的新的调度策略，在此调度策略下：
+* 用户可以在 Policy 中对这个集群指定优先级
+* 对于不同优先级的集群，副本优先分配到高优先级的集群，只有在高优先级集群资源不足后，副本才会分配给低优先级集群
+* 对于相同优先级的集群，将副本划分到尽可能少的集群
+-->
+
+This section introduces a new scheduling strategy called `PriorityAggregated`:
+
+* Users can specify the priority of clusters in the policy.
+* Replicas are allocated to higher-priority clusters first, 
+  only when resources are insufficient in those clusters will replicas be assigned to lower-priority clusters.
+* For clusters with the same priority, replicas are divided into as few clusters as possible.
+
+#### API changes
+
+add a new `ReplicaDivisionPreference` type named `PriorityAggregated`
+
+```go
+// ReplicaDivisionPreference describes options of how replicas can be scheduled.
+type ReplicaDivisionPreference string
+
+const (
+    ReplicaDivisionPreferenceAggregated ReplicaDivisionPreference = "Aggregated"
+    ReplicaDivisionPreferenceWeighted ReplicaDivisionPreference = "Weighted"
++	// ReplicaDivisionPreferencePriorityAggregated assigns replicas to higher-priority clusters first, 
++	// then to lower-priority clusters if resources are insufficient.
++	ReplicaDivisionPreferencePriorityAggregated ReplicaDivisionPreference = "PriorityAggregated"
```

Comment:
```suggestion
	// ReplicaDivisionPreferencePriorityAggregated assigns replicas to higher-priority clusters first, 
	// then to lower-priority clusters if resources are insufficient.
	ReplicaDivisionPreferencePriorityAggregated ReplicaDivisionPreference = "PriorityAggregated"
```

Reply:
Does the `PriorityAggregated` still honor the score plugins in the scheduler? In another world, what's the relationship of them?